### PR TITLE
Added doc steps for EFS configuration

### PR DESCRIPTION
**Pull request by ArvinthC3000**
Signed-off-by: Arvinth C <arvinth.chandrasekaran@progress.com>

### :nut_and_bolt: Description: What code changed, and why?
In this PR, a new section of Configuring Automate to register the snapshot locations with OpenSearch is detailed with some modifications on EFS backup.

### :chains: Related Resources

Jira ticket: https://chefio.atlassian.net/browse/MADROX-240

### :+1: Definition of Done
It is dev done and tested

### :white_check_mark: Checklist

**All PRs** must tick these:

- [ ] I have read the [CONTRIBUTING document](https://github.com/chef/automate/blob/master/CONTRIBUTING.md).
- [ ] All commits signed-off for the [Developer Certification of Origin](https://github.com/chef/chef/blob/master/CONTRIBUTING.md#developer-certification-of-origin-dco).

With occasional exceptions, all PRs **from Progress employees** must tick these:

- [ ] Is the code clear? *(complicated code or lots of comments--subdivide and use well-named methods, meaningful variable names, etc.)*
- [ ] Consistency checked? *(user notifications, user prompts, visual patterns, code patterns, variable names)*
- [ ] Repeated code blocks eliminated? *(adapt and reuse existing components, blocks, functions, etc.)*
- [ ] Spelling, grammar, typos checked? *(at a minimum use `make spell` in any component directory)*
- [ ] Code well-formatted? *(indents, line breaks, etc. improve rather than hinder readability)*

All PRs **from Progress employees** should tick these if appropriate:

- [ ] Tests added/updated? (all new code needs new tests)
- [ ] Docs added/updated? (all customer-facing changes)

*Please add a note next to any checkbox above if you are NOT ticking it.*

### CODE REVIEW

File: components/docs-chef-io/content/automate/ha_backup_restore_prerequisites.md [hunk 1]
```diff
@@ -105,6 +105,7 @@ Refer to the content for the `automate.toml` file below:
     name = "bucket-name"
 
     # endpoint (required): The endpoint for the region the bucket lives in.
+    # See https://docs.aws.amazon.com/general/latest/gr/rande.html#s3_region
```
Comment (vivek-yadav) [inline components/docs-chef-io/content/automate/ha_backup_restore_prerequisites.md:108]: remove this line

Comment (ArvinthC3000) [reply components/docs-chef-io/content/automate/ha_backup_restore_prerequisites.md]: Done

File: components/docs-chef-io/content/automate/ha_backup_restore_prerequisites.md [hunk 2]
```diff
@@ -130,30 +130,102 @@ Take a [back-up](/automate/ha_restore/) of the configurations once the cluster h
...
+```sh
+[path]
+# Replace /mnt/automate_backups with the backup_mount config found on the provisioning host in /hab/a2_deploy_workspace/a2ha.rb
+repo = "/mnt/automate_backups/opensearch"
+```
+
+To trigger the restart of the OpenSearch on each server, apply the updated `es_config.toml` config to OpenSearch once.
+
+```sh
+hab config apply automate-ha-opensearch.default $(date '+%s') es\_config.toml
```
Comment (punitmundra) [inline components/docs-chef-io/content/automate/ha_backup_restore_prerequisites.md:170]: es\_config.toml -> es_config.toml

Comment (ArvinthC3000) [reply components/docs-chef-io/content/automate/ha_backup_restore_prerequisites.md]: Done

File: components/docs-chef-io/content/automate/ha_backup_restore_prerequisites.md [hunk 3]
```diff
@@ -14,22 +14,22 @@ gh_repo = "automate"
 +++
 
 This page explains the prerequisites of the backup.
-If We choose AWS Deployment Procedure to deploy Automate-HA and we have choosen the `backup_config = efs OR s3` 
-in `config.toml`, in that case below steps are not required. Below steps are taken care by the deployment. 
-If we have kept the `backup_config` blank, in that case to need to perform the below steps. 
+If We choose AWS Deployment Procedure to deploy Automate-HA and we have choosen the `backup_config = efs OR s3`
```
Comment (vivek-yadav) [inline components/docs-chef-io/content/automate/ha_backup_restore_prerequisites.md:17]:
```suggestion
If We choose AWS Deployment Procedure to deploy Automate-HA and we have choosen the `backup_config = "efs" OR "s3"`
```

**Review by dishanktiwari2501 — APPROVED** [review]

**Review by dishanktiwari2501 — APPROVED** [review]

**Review by vivek-yadav — APPROVED** [review]

**Review by jayvikramsharma1 — APPROVED** [review]

**Review by punitmundra — APPROVED** [review]

**Pull request merged**